src/TypeScript.UnitTests/TsConfig/TsConfigProviderTests.cs

```diff
@@ -66,24 +66,24 @@ public async Task GetConfigForFile_NoTsConfigsInSolution_Null()
         public async Task GetConfigForFile_SourceFileIsNotInAnyTsConfig_Null()
         {
             const string testedFileName = "some file";
-            var tsConfigsInSolution = new[] { "config1", "config2", "config3" };
+            var tsConfigsInSolution = new[] { "config1.json", "config2.json", "config3.json" };
```

Comment:
I had to add an extension to all the existing tests, otherwise they were considered as a folder due to the new logic.